app/assets/stylesheets/aspects.scss

```diff
@@ -21,3 +21,7 @@
     }
   }
 }
+.aspect_membership {
```

Comment:
Selector `aspect_membership` should be written in lowercase with hyphens

Reply:
When I change this to aspect-membership several cucumber and Jasmine tests fail.  One Jasmine test it was very straight forward since it was referencing it directly. The others I can't tell why the style change would create a change in behavior.  The errant behavior that is being flagged, such as not getting the notification when adding a person to the aspect or mis-count of aspects, doesn't seem to be happening in the User interface and again I'm not sure why changing the style name and its subsequent references throughout is creating the test failures.

Reply:
It's OK to ignore these, if it's not just a small change at a few places. Sometimes stuff like this needs to be changed in many places because it's used everywhere (in tests to select stuff, in js to select stuff, at other places in css, ...), and then it isn't worth to do it. If you still like to change this, please do it in a separate commit :)